opm/input/eclipse/Schedule/Well/WListManager.hpp

```diff
@@ -53,21 +68,135 @@ class WListManager
     /// pattern.
     bool hasWell(const std::string& pattern) const;
 
-    bool hasList(const std::string&) const;
+    /// Predicate for existence of particular well lists
+    ///
+    /// \param[in] wlistName Well list name, including leading asterisk.
+    ///
+    /// \return Whether or not well list named \p wlistName exists in
+    /// current collection.
+    bool hasList(const std::string& wlistName) const;
+
+    /// Access individual well list by name.
+    ///
+    /// Mutable version.
+    ///
+    /// Will throw an exception if the the well list does not exist.  Use
```

Comment:
the the

Reply:
> the the

Doh!  I've pushed an update to address this.